docs/connect.html

```diff
@@ -58,10 +58,13 @@
 <ul>
     <li><code>group.id</code> (default <code>connect-cluster</code>) - unique name for the cluster, used in forming the Connect cluster group; note that this <b>must not conflict</b> with consumer group IDs</li>
     <li><code>config.storage.topic</code> (default <code>connect-configs</code>) - topic to use for storing connector and task configurations; note that this should be a single partition, highly replicated topic</li>
-    <li><code>offset.storage.topic</code> (default <code>connect-offsets</code>) - topic to use for ; this topic should have many partitions and be replicated</li>
+    <li><code>offset.storage.topic</code> (default <code>connect-offsets</code>) - topic to use for storing offsets; this topic should have many partitions and be replicated</li>
+    <li><code>status.storage.topic</code> (default <code>connect-status</code>) - topic to use for storing statuses; this topic can have multiple partitions and be replicated</li>
 </ul>
 
-Note that in distributed mode the connector configurations are not passed on the command line. Instead, use the REST API described below to create, modify, and destroy connectors.
+We recommend to manually create the topics for offset, configs and statuses in order to achieve the desired the number of partitions and replication factor. If the topics are not yet created when starting Kafka Connect, the topics will be auto created with default number of partitions and replication factor, which may not be best suited for its usage. 
```

Comment:
What would be the impact of creating the config storage topic with more than one partition? I looked in the code and didn't see any assertions that it actually has just one partition, so it seems like it might sort of work but with all of its config offset expectations invalidated? Maybe we should just tell users they MUST create topics manually since they're unlikely to have set a default number of partitions to 1? Also maybe we should assert on startup that the config topic has exactly one partition.